assets/scss/layouts/_sidebar.scss

```diff
@@ -85,13 +85,36 @@ a.docs-link {
   font-size: $font-size-base * 0.9375;
 }
 
+.docs-link {
+  transition: all 0.3s;
+}
```

Comment:
Sorry to dig up an old PR, and sorry for the silly question. Why is there a transition here? Is this strictly necessary to add the dot at the start of the current menu item? Or does it affect something else not related to the dot?